src/serializer.ts

```diff
@@ -45,10 +42,10 @@ export class Serializer {
     public register(registration: Registration[]): void {
         for (const reg of registration) {
 
-            const parentOption: ParentOptions = Serializer.getParentOptions(reg.parent);
+            const parentOption = Serializer.getParentOptions(reg.parent);
```

Comment:
might be parentOptions* but I don't know why I didn't name it like this by my own...